src/copy.c

```diff
@@ -218,8 +218,7 @@ timescaledb_CopyFrom(CopyState cstate, Relation main_rel, List *range_table, Hyp
 		Oid			loaded_oid = InvalidOid;
 		Point	   *point;
 		ChunkDispatch *dispatch = ccstate->dispatch;
-		ChunkInsertState *cis,
-				   *prev_cis = NULL;
```

Comment:
The initialization of `prev_cis` should be moved outside the loop to allow reuse of the `BulkInsertState` to be reused for the same chunk.